images/tensorflow-notebook/cuda12/Dockerfile

```
@@ -0,0 +1,34 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+ARG REGISTRY=quay.io
+ARG OWNER=jupyter
+ARG BASE_CONTAINER=$REGISTRY/$OWNER/scipy-notebook
+FROM $BASE_CONTAINER
+
+LABEL maintainer="Jupyter Project <jupyter@googlegroups.com>"
+
+# Fix: https://github.com/hadolint/hadolint/wiki/DL4006
+# Fix: https://github.com/koalaman/shellcheck/wiki/SC3014
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Install CUDA libs and cuDNN with mamba
+RUN mamba install --yes -c nvidia/label/cuda-12.3.2 \
+    'cuda-libraries-dev' \
+    'cuda-nvcc' \
+    'cuda-nvtx' \
+    'cuda-cupti' \
+    'cudnn' && \
```

```suggestion
    'cuda-cupti' \
    'cuda-libraries-dev' \
    'cuda-nvcc' \
    'cuda-nvtx' \
    'cudnn' && \
```

Let's keep the list sorted